cmd/collection/main.go

```diff
@@ -117,6 +121,10 @@ func main() {
 				"additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
+
+			// epoch qc contract flags
+			flags.StringVar(&accessAddress, "access-address", "DEFAULT_ACCESS_ADDRESS?", "the address of an access node")
+			flags.StringVar(&qcContractAddress, "qc-contract-address", "DEFAULT_QC_CONTRACT_ADDRESS?", "the address of the Epoch QC contract")
```

Comment:
I don't think a default makes sense for these and I don't see a way to force a non-default input for the flag library we're using, so I would set the defaults here to the empty string and check for this while constructing the `QCContractClient`.
```suggestion
			flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
			flags.StringVar(&qcContractAddress, "qc-contract-address", "", "the address of the Epoch QC contract")
```